### PR TITLE
Added logger logic to project

### PR DESCRIPTION
**Pull request by Biskota**

### CODE REVIEW

File: src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
+            </Filters>
         </Console>
         <File name="File" fileName="demoApplication.log" append="false">
             <PatternLayout pattern="%d{HH:mm:ss} [%t] %-5level %logger{36} - %msg%n"/>
+            <Filters>
+                <ThresholdFilter level="warn"  onMatch="ACCEPT" onMismatch="NEUTRAL"/>
+                <ThresholdFilter level="error" onMatch="ACCEPT" onMismatch="NEUTRAL"/>
+                <ThresholdFilter level="fatal" onMatch="ACCEPT" onMismatch="NEUTRAL"/>
+
+                <ThresholdFilter level="info"  onMatch="DENY" onMismatch="DENY"/>
```
Comment (DashaLevchenko) [inline src/main/resources/log4j2.xml:22]: without info in file

File: src/main/java/menu/TaskManagerCommands.java [hunk 2]
```diff
@@ -1,41 +1,44 @@
...
 
 import entity.Task;
+import org.apache.logging.log4j.*;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
 
 public class TaskManagerCommands {
+    static final Logger logger = LogManager.getRootLogger();
```
Comment (4etverochka) [inline src/main/java/menu/TaskManagerCommands.java:11]: Better to use construction `private static final Logger LOGGER = LogManager.getLogger(TaskManagerCommands.class.getName());`

File: src/main/java/menu/TaskManagerCommands.java [hunk 3]
```diff
@@ -1,41 +1,44 @@
...
-        System.out.println("remove");
+        logger.info("remove");
     }
 
     public void createTask() {
-        System.out.println("Task title:");
+        logger.info("Task title:");
         String title = scanConsoleInput();
-        System.out.println("Task description:");
+
```
Comment (4etverochka) [inline src/main/java/menu/TaskManagerCommands.java:35]: Delete unnesessary enter

File: src/main/java/menu/WriterTasks.java [hunk 4]
```diff
@@ -8,34 +8,40 @@
 import java.time.format.DateTimeFormatter;
 import java.util.*;
 
+import org.apache.logging.log4j.*;
+
+
 public class WriterTasks {
     private static String fileName;
+    static final Logger logger = LogManager.getRootLogger();
```
Comment (4etverochka) [inline src/main/java/menu/WriterTasks.java:16]: Same there about initializing logger

File: src/main/java/menu/TasksManager.java [hunk 5]
```diff
@@ -1,17 +1,19 @@
 package menu;
 
+import org.apache.logging.log4j.*;
+
 import static menu.TaskManagerCommands.scanConsoleInput;
 
 public class TasksManager {
+    static final Logger logger = LogManager.getRootLogger();
```
Comment (4etverochka) [inline src/main/java/menu/TasksManager.java:8]: Same there about initializing logger

File: src/main/java/menu/WriterTasks.java [hunk 6]
```diff
@@ -8,34 +8,40 @@
...
-            //log error message
+            logger.error("File not found", e);
         } catch (IOException e) {
             e.printStackTrace();
-            //log error message
+            logger.error("Something wrong with file", e);
         } catch (ClassNotFoundException e) {
             e.printStackTrace();
+            logger.error("Class not found", e);
+
```
Comment (4etverochka) [inline src/main/java/menu/WriterTasks.java:44]: Delete unnesesary enter

File: src/main/java/runner/Run.java [hunk 7]
```diff
@@ -7,5 +7,6 @@
     public static void main(String[] args) {
         TasksManager tasksManager = new TasksManager();
         tasksManager.menu();
+
```
Comment (4etverochka) [inline src/main/java/runner/Run.java:10]: Same there

File: src/main/java/menu/WriterTasks.java [hunk 8]
```diff
@@ -8,34 +8,40 @@
...
+    static final Logger logger = LogManager.getRootLogger();
 
-    public static void serializeTasks(List<Task> tasks)  {
+    public static void serializeTasks(List<Task> tasks) {
         File file = rename(getFile());
         try (ObjectOutputStream out = new ObjectOutputStream(new FileOutputStream(file))) {
             out.writeObject(tasks);
         } catch (FileNotFoundException e) {
-            //log error message
+            logger.error("File not found", e);
```
Comment (4etverochka) [inline src/main/java/menu/WriterTasks.java:23]: I think we need to log only message

**Comment by DashaLevchenko** [comment]
Ок

File: src/main/java/menu/TaskManagerCommands.java (hunk 2 above)
Comment (4etverochka) [reply src/main/java/menu/TaskManagerCommands.java]: @Biskota look at this comment and try to inplement

**Pull request merged**